src/Apps/Artists/Routes/ArtistsByLetter.tsx

```diff
@@ -145,7 +145,7 @@ const buildH1Title = (letter: string, page: number): string => {
   const baseTitle = `Artists - ${letter.toUpperCase()}`
   const isPagedContent = page > 1
 
-  return isPagedContent ? `Page ${page}: ${baseTitle}` : baseTitle
+  return isPagedContent ? `${baseTitle} - Page ${page}` : baseTitle
```

Comment:
```suggestion
  return isPagedContent ? <>{baseTitle}<VisuallyHidden> - Page {page}</VisuallyHidden></> : baseTitle
```

Reply:
👍 I implemented this, and tool the opportunity to use `MetaTags` for this surface as well.